File: Concurs Inot Java/Proba.java
package me.model;

import java.io.Serializable;

public class Proba extends Entity<Integer> implements Serializable {
    private float distanta;
    private Stiluri stil;

    public Proba(Integer id, float distanta, Stiluri stil) {
        super.setId(id);
        this.distanta = distanta;
        this.stil = stil;
    }

    public float getDistanta() {
        return distanta;
    }

    public void setDistanta(float distanta) {
        this.distanta = distanta;
    }

    public Stiluri getStil() {
        return stil;
    }

    public void setStil(Stiluri stil) {
        this.stil = stil;
    }

    @Override
    public String toString() {
        return "proba " + distanta + "m "+ stil;
    }
}

File: Concurs Inot C#/Proba.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problema_3_CSharp.me.model
{
    [Serializable]
    public enum Stiluri { liber, spate, fluture, mixt }
    [Serializable]
    public class Proba : Entity<int>
    {
        public Stiluri Stil { get; set; }
        public float Distanta { get; set; }
        public Proba(int id, float distanta, Stiluri stil)
        {
            base.Id = id;
            Distanta = distanta;
            Stil = stil;
        }

        public override string ToString()
        {
            return Distanta + "m " + Stil;
        }
    }
}
